### ISSUE TITLE
The projects' name is white on "High Contrast on" mode

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-microbit#922]
**Repro steps:**
• Navigate to https://makecode.microbit.org/beta#editor
• Click "More..." -> "High Contrast On" to open "High Contrast On" mode
• Click "Home" to back to home page
• Click any project and observe the dialog

**Expected result:**
The project name should be other color to show more clearly

**Actual result:**
The project name's name is white and it is not clear enough
![issue70503](https://user-images.githubusercontent.com/23466737/42316383-33d5a0b0-807c-11e8-9d33-58a66e4d3572.PNG)

**Comment by AimeeCao18** [comment]
The bug still repro now

**Comment by Helen09** [comment]
Tutorial name and background luminosity ratio is less than 4.5:1 on high contrast on mode
![issue2210121](https://user-images.githubusercontent.com/23466737/53154672-43f70580-35f6-11e9-9e5b-60042185068f.PNG)

**Comment by Helen09** [comment]
This bug still repro on high contrast mode
